.circleci/config.yml

```diff
@@ -61,7 +61,7 @@ jobs:
     working_directory: ~/app
     docker:
       - image: cimg/node:<< pipeline.parameters.node-version >>
-      - image: cimg/postgres:14.0
+      - image: cimg/postgres:14.3
```

Comment:
This doesn't seem to be related with the db refresh job. Does it need to be changed (as it is already 14.0)?